src/mono/cmake/configure.cmake

```diff
@@ -160,7 +160,7 @@ check_c_source_compiles(
   #include <sched.h>
   int main(void)
   {
-    CPU_COUNT((void *) 0);
+    CPU_COUNT((cpuset_t *) 0);
```

Comment:
Building Mono is also not necessary to get a working NativeAOT setup.

Reply:
This will be reverted as its addressed in another PR